lua/nvim-tree/lib.lua

```diff
@@ -81,6 +85,8 @@ function M.get_last_group_node(node)
   return next_node
```

Comment:
I really don't get why it complains about a missing nil check here... And consequently a possible nil return